src/nuis/measurement/ProjectionHelper.cxx

```diff
@@ -217,7 +212,7 @@ int Record::GetBin(const std::vector<double>& invals) {
     return -1;
 }
 
-int Record::FillBin(int i, double w) {
+int ProjectionHelper::FillBin(int i, double w) {
```

Comment:
why does this return the argument or -1. If it return -1 for underflow and nbins for overflow, that would be okay, but as is its probably better to make it return a bool rather than just the argument modified only in error case.